oak/server/enclave_server.h

```diff
@@ -58,22 +57,28 @@ class EnclaveServer final : public asylo::TrustedApplication {
  public:
   EnclaveServer()
       : node_(nullptr),
-        credentials_(
-            ::asylo::EnclaveServerCredentials(::asylo::BidirectionalNullCredentialsOptions())),
+        credentials_(::asylo::EnclaveServerCredentials(
```

Comment:
Not sure why this was reformatted this way? maybe we are using different versions of clang-format? FTR, I am using the internal Google one (clang-format version google3-trunk (trunk r357757)), not sure if the external one is different?

Reply:
I thought I was using the same one but apparently it had a different setting for line width; will check it on my end